.github/workflows/tests.yml

```diff
@@ -9,10 +11,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.8
+    - name: Set up Python 3.12
       uses: actions/setup-python@v2
       with:
-        python-version: 3.8
+        python-version: 3.12
```

Comment:
Please add:
* 3.10 too, we use 3.10 in Connect
* 3.11 - that use HT90, which will use PrusaErrroCodes too

Reply:
Added matrix configuration to run with multiple Pythons.